sdk/core/azure-core-test/inc/azure/core/test/test_context_manager.hpp

```diff
@@ -20,6 +20,11 @@ namespace Azure { namespace Core { namespace Test {
    */
   class TestContextManager {
   public:
+    /**
+     * @brief The path where the asset.json exists.
+     *
+     */
+    std::string AssetsPath;
```

Comment:
Why is the AssetsPath on both the TestBase and the TestContextManager?  Can you add more info to the brief explaning how this value flows between the two.

Reply:
the test context uses the assets path in order to put toghether the start playback/record request. the idea is that you can override the GetAssetsPath to another location in the service testbase implementation. When the test context is initialized the assets path is set , then later on cosumed when each test is set to run, for each test you need to call startplayback , with the test name and the assets file